test-requirements.txt

```diff
@@ -3,3 +3,4 @@ testfixtures >= 6.10.3
 pytest-env >= 0.6.0
 pytest-xdist >= 2.0
 flaky >= 3.0
+requests-mock >= 1.9.3
```

Comment:
Generally, I'm quite hesitant to add testing requirements for contributions to the task library. I'm not sure this gives us much more than a simple mock, but I'm willing to leave it in.

Reply:
Interestingly, another user wrote a task library PR and is using `responses` for mocking. https://github.com/getsentry/responses

At a glance, I think that project looks more interesting although it's pre-1.0. Curious if you have any thoughts.